### PR TITLE
Om-ify the website

### PR DESCRIPTION
**Pull request by expede**
Gives us several advantages:
- Sequencing animations
- Coordinated state
- Extensibility
- Easily integrated with front-end routers, such as [Secretary](https://github.com/gf3/secretary)

Still need to animate the staff and contact sections, but they're waaaaay below the fold. I think that this is eligible to go into `master` now, and we can add the fancy CSS & visibility detection later.

Issue Link: #2

### CODE REVIEW

**Comment by expede** [comment]
@jenncoop for the code review: to run, you should only need to open the `resources/public/index.html` page in a browser. No need to run a server or anything.

**Comment by expede** [comment]
Also, it's not _really_ 113 files to review. I renamed a directory :wink:

**Comment by jenncoop** [comment]
@expede Haha, kk, will review... :smile_cat:

File: src/robot_overlord/core.cljs [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+            [om.dom  :as dom :include-macros true]))
+
+(enable-console-print!)
+
+;; =====
+;; UTILS
+;; =====
+(defn start-cycle
+  "Start the render cycle"
+  [owner]
```
Comment (jenncoop) [inline src/robot_overlord/core.cljs]: Where does `owner` come from?

File: src/robot_overlord/core.cljs [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+  (om/refresh! owner))
+
+(defn make-visible!
+  "Set visibiloty state of some element to true"
```
Comment (jenncoop) [inline src/robot_overlord/core.cljs]: Small typo here

**Comment by jenncoop** [comment]
Once source gets pushed up, you're good to merge!!! :+1:

**Comment by jenncoop** [comment]
Once source gets pushed up, you're good to merge!!! :+1: 

shipships

**Pull request merged**